doc/7-systems/pokemini.md

```diff
@@ -0,0 +1,7 @@
+# Pokémon mini
```

Comment:
Capitalize first letter of Mini.

Reply:
https://www.nintendo.co.uk/Hardware/Nintendo-History/Pokemon-mini/Pokemon-mini-627132.html

Reply:
Sorry but I will capitalize Mini.